src/core_plugins/kibana/public/dashboard/panel/panel.html

```diff
@@ -1,4 +1,11 @@
-<div class="panel panel-default" ng-switch on="panel.type" ng-if="savedObj || error">
+<div
+  class="panel panel-default"
+  ng-switch on="panel.type"
+  ng-if="savedObj || error"
+  reporting-item
```

Comment:
Good point, I initially moved it here when attempting to resolve the issue with gridster incrementally adding the dashboard-panels, but I agree it should be moved back for consistency.